mathics/packages/ob-mathics.el

```diff
@@ -0,0 +1,137 @@
+;;; ob-mathics.el --- org-babel functions for mathics evaluation
+
+;; Copyright (C) Brian Beckman
```

Comment:
You cannot claim the rights for the contents of the files you submit in a PR. They must be licensed according to [our licensing scheme](https://github.com/mathics/Mathics/blob/master/COPYING.txt).

Reply:
Eventhought you are using GLPv3, we cannot accept code with the comment `Copyright (C) Brian Beckman`.